job_spec/VOLCSARVATORY_MULTI_BURST.yml

```
@@ -0,0 +1,78 @@
+VOLCSARVATORY_MULTI_BURST:
```

```suggestion
INSAR_ISCE_MULTI_BURST:
```
I'd keep this name the same so that it will just be the multiburst job in the volcsarvatory deployment